projects/plugins/jetpack/modules/seo-tools/class-jetpack-seo-utils.php

```diff
@@ -41,13 +39,9 @@ public static function is_enabled_jetpack_seo( $site_id = 0 ) {
 			return false;
 		}
 
-		if ( function_exists( 'has_any_blog_stickers' ) ) {
-			// For WPCOM simple sites.
-			if ( empty( $site_id ) ) {
-				$site_id = get_current_blog_id();
-			}
-
-			return has_any_blog_stickers( array( 'business-plan', 'ecommerce-plan' ), $site_id );
+		// For WPcom sites.
```

Comment:
I believe you can actually retain the previous comment here instead ("For WPCOM **simple** sites")?

It looks to me like your code does preserve the existing behavior of always returning `true` for Atomic sites, which is good.